examples/gradle-and-jvm/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Fri Jun 05 19:21:53 PDT 2020
```

Comment:
The gradle wrapper files have been intentionally left out of the example projects to make it a bit easier to keep them in sync with the main project. You can just run `../../gradlew` from one of the examples to use the main project's gradle wrapper.

I could be convinced to add the gradle wrapper to each example project. But if that's something you think we should do, let's have that conversation in a separate PR.

Reply:
These got added accidentally I think because I imported the project into Intelij.